cloud-config/aws/README.md

```diff
@@ -0,0 +1,156 @@
+
+## Prerequisites
+
+**Warning**: Make sure at all time that you have selected the availability zone where you want to perform your actions. You can choose the availability zone using the dropdown menu at the top-right corner of the AWS console website.
+
+### Create a VPC
+
+- Go to the [VPC Dashboard](https://console.aws.amazon.com/vpc/home#dashboard)
+  
+  Click on `Launch VPC Wizard`.
+
+- Configure the new VPC
+
+  Follow the wizard for the `VPC with a Single Public Subnet` configuration.
+  
+  Give your VPC a name setting a value in the `VPC Name` field.
+  
+  All the defaults are OK. 
+  
+  Click on `Create VPC`.
+  
+### Create a security group
+
+- Go to the [VPC - Security groups panel](https://console.aws.amazon.com/vpc/home#SecurityGroups:sort=groupId)
+
+  Click on `Create security group`.
+
+- Create the new security group
+
+  Set a name and a description for your new security group.
+  
+  Select the VPC you've created in the previous step.
+  
+  Click on `create`.
+- Set the inbound rules in the security group
+
+  Click on the newly created security group from the list.
+
+  Click on the `Inbound rules` tab, and click on the `Edit rules` button.
+
+  Add the following rules:
+  
+  | Type | Source |
+  | --- | --- |
+  | SSH | Anywhere |
+  | HTTP | Anywhere |
+  | HTTPS | Anywhere |
+  
+  Click on `Save rules`.
+
+## Create an IAM role
+
+- Go to the [IAM - Roles panel](https://console.aws.amazon.com/iam/home#/roles)
+
+
+  Click on `Create role`.
+- Configure the new role
+
+  Make sure that the `AWS service` box is selected, and click on the `EC2` link.
+  
+  Click on `Next: Permissions`.
+- Add policies to the role
+
+  Search for `AmazonEC2ReadOnlyAccess`, and select it.
+  
+  Click on `Next: Tags`. (nothing to do here)
+  
+  Click on `Next: Review`.
+- Give the new IAM role a name
+
+  Click on `Create role`.
+
+## Create the EC2 machine
+
+- Go to the [EC2 Dashboard](https://console.aws.amazon.com/ec2/v2/home#Home:)
+  
+  Click on `Launch instance`
+- Search for the `Ubuntu Server 18.04` AMI
+
+  You should select the AMI showing `Ubuntu Server 18.04 LTS (HVM), SSD Volume Type` tag line.
+  
+  Make sure you select the `64-bit (x86)` option under the `Select` button.
+  
+  Click on `Select`.
+- Select the instance type you want to use.
+
+  Click on `Next: Configure Instance Details`
+
+- Set the instance details.
+
+  Select the VPC you've created in the `Network` dropdown.
+  
+  Select `Enable` in the `Auto-assign Public IP` dropdown.
+  
+  Select the IAM role you've created in the previous step from the `IAM role` dropdown.
+  
+  Toggle the `Advanced Details` section.
+  
+  Copy the contents of the [Aggregate Cloud Config stack for AWS](https://raw.githubusercontent.com/opendatakit/aggregate/master/cloud-config/aws/cloud-config.yml) into the `User data` text box.
+
+  Click on `Next: Add Storage`.
```

Comment:
If we pick a reasonable size machine (e.g.,  match the machine size on DO?) then this step goes away.

Reply:
Most of the instance types are EBS only storage (especially t1, including the free tier type), which requires to add a storage volume. 

The instance types that include storage are kind of expensive, but maybe we should add something like "If you select an instance type with built-in storage, you can skip this section."

Reply:
You can customize the EBS volumes with bigger root drives, no? Does that have some downside I'm not thinking of?

Reply:
EBS volumes are fully customizable, and you get to have snapshots and even scale your instance up or down without having to reinstall software or migrate data.

I've seen how AWS has been progressively moving away from built-in storage to EBS only storage, at least on basic and general purpose instance types.

For specific high IO applications, there are still instance types that come with built-in specialized storage.